downstream/modules/platform/con-why-automation-mesh.adoc

```diff
@@ -16,6 +16,6 @@ The {AutomationMesh} component of the {PlatformName} simplifies the process of d
 
 * For information about automation mesh and the various ways to design your automation mesh for your environment:
 
-** For a VM-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_automation_mesh_guide_for_vm-based_installations/index[{PlatformName} {AutomationMesh} guide for VM-based installations].
+** For a VM-based installation, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_mesh_for_vm_environments/index[{PlatformName} {AutomationMesh} guide for VM-based installations].  
```

Comment:
The title of this document is now "Automation mesh for VM environments" and Aine set up attributes for URLs and links, suggest changing this to use that attribute:

link:{LinkAutomationMesh}

Reply:
Still getting used to this! Thanks.